actions/changed-files/check_changed_files.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# THIS expects two environment variables to be present
+#  - INCLUDE_PATTERN
+#  - EXCLUDE_PATTERN
+
+set -ex
+
+if [ -z "${GITHUB_BASE_REF}" ]; then
+  # this is not a pull request event, just get the previous commit
+  previous_sha=$(git rev-parse --verify 'HEAD^{commit}')
+else
+  previous_sha=${GITHUB_BASE_REF}
+fi
+CHANGED_FILES=$(git diff --name-only --diff-filter=ACMRT ${previous_sha}..HEAD )
+
+if [ -z "${EXCLUDE_PATTERN}" ]; then
+  echo "${CHANGED_FILES}"
+  exit 0
+fi
```

Comment:
`EXCLUDE_PATTERN` can now be empty, but we still need to match files against the patterns in `INCLUDE_PATTERNS`.

We need do move this to the for cycle below, replacing the exit with a break?

Reply:
I missed to removed this code . The condition is already in place